### DISCUSSION TITLE
How can I use official player on extra version

### DISCUSSION THREAD

**Discussion by DahilerMehvOlur** [SkyTubeTeam/SkyTube#1285] [Q&A]
Hello. When I select official player in settings it gives me error and it says "Official youtube player cannot run"
I think it is not youtube app. Must be different thing. But what? Can any one clarify me? What I need for use official player?

**Comment by gzsombor** [comment]
No, the official youtube player is indeed - or was - the offical, embeddable Youtube player component by Google - but it was not supported by anymore: https://developers.google.com/youtube/android/player . The library is hosted [here](https://github.com/SkyTubeTeam/SkyTube/blob/master/app/libs/YouTubeAndroidPlayerApi.jar) - as you can see, it's pretty old, added 8 years ago. 
I've never used it personally, some users preferred to use it, maybe on some specific old Androids it behaved better than the other players.

**Reply by DahilerMehvOlur** [reply]
Ah ok. Everything is clear now. Thanks.
